.stylelintrc

```diff
@@ -1,5 +1,5 @@
 {
-	"extends" : "stylelint-config-wordpress/scss",
```

Comment:
This is deprecated, see https://www.npmjs.com/package/stylelint-config-wordpress